packages/block-library/src/navigation-submenu/index.php

```diff
@@ -188,6 +188,12 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 		);
 	}
 
+	$arialabel = sprintf(
```

Comment:
Nitpick: Maybe we should use underscore as a separator in name of this variable?

Reply:
Yes, the variable should be `$aria_label`.

Reply:
Updated in the 2 latest commits! Thanks for pointing out!